### PR TITLE
kubeadm: Upgrade Bootstrap Tokens to beta when upgrading to v1.8

### PR DESCRIPTION
**Pull request by luxas**
**What this PR does / why we need it**:

Makes sure the v1.7 -> v1.8 upgrade works regarding the Bootstrap Token alpha -> beta graduation.
Not much have to be done, but some LoC are needed to preserve the behaivor

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```
@kubernetes/sig-cluster-lifecycle-pr-reviews

### CODE REVIEW

**Review by jbeda — APPROVED** [review]

**Comment by dims** [comment]
@jbeda - this needs a `/approve no-issue` please

**Comment by dims** [comment]
@luxas looks like gofmt problems in verify. PTAL

**Comment by luxas** [comment]
@jbeda PTAL.
In v1.7 we used `system:bootstrap-signer-clusterinfo` as the Role name for cluster-info.
This PR now deletes that Role as well (it's now created as `kubeadm:bootstrap-signer-clusterinfo`), and deletes the associated RoleBinding so that it can be recreated with the right roleRef.

cc @mattmoyer

File: cmd/kubeadm/app/constants/constants.go [hunk 1]
```diff
@@ -123,8 +123,7 @@ const (
 	KubeConfigVolumeName = "kubeconfig"
 
 	// NodeBootstrapTokenAuthGroup specifies which group a Node Bootstrap Token should be authenticated in
-	// TODO: This should be changed in the v1.8 dev cycle to a node-BT-specific group instead of the generic Bootstrap Token group that is used now
-	NodeBootstrapTokenAuthGroup = "system:bootstrappers"
+	NodeBootstrapTokenAuthGroup = "system:bootstrappers:kubeadm:default-node-token"
```
Comment (timothysc) [inline cmd/kubeadm/app/constants/constants.go]: What happens if you use a kubeadm 1.8 version to deploy a 1.7 cluster will this work?

**Comment by jbeda** [comment]
/approve no-issue

Please address comment from Tim before lgtm.

**Comment by roberthbailey** [comment]
Please make sure kubeadm released with 1.8 still supports deploying 1.7 clusters as they are done today (e.g. with the same token names).

**Comment by luxas** [comment]
/retest

@mattmoyer @timothysc PTAL
This now works with v1.7 as well, thanks for spotting!

**Review by timothysc — APPROVED** [review]
/lgtm

**Pull request merged**